cpp/benchmarks/join/join_benchmark.cu

```diff
@@ -46,11 +46,32 @@ static void BM_join(benchmark::State &state)
   const bool is_build_table_key_unique = true;
 
   // Generate build and probe tables
-
-  auto build_key_column =
-    cudf::make_numeric_column(cudf::data_type(cudf::type_to_id<key_type>()), build_table_size);
-  auto probe_key_column =
-    cudf::make_numeric_column(cudf::data_type(cudf::type_to_id<key_type>()), probe_table_size);
+  std::srand(1337);
```

Comment:
We have random number generation utilities in cudf_test for use in tests and benchmarks. See `reduce_benchmark.cpp` for example.

This is just a suggestion, I'm not sure what exact range you should use.

```suggestion
   cudf::test::UniformRandomGenerator<size_type> rand_gen(0, build_table_size);
```